spec/redux/reducer-test.js

```diff
@@ -0,0 +1,36 @@
+import loadApi from '../../src/redux/reducer';
+import * as actType from '../../src/redux/action-types';
+
+describe("The reducer", () => {
+
+  it("returns a state object", () => {
+    expect(loadApi({}, "default")).toEqual(jasmine.any(Object));
+  });
+
+  describe("returns a state change", () => {
+
+    var state = {
+      loadingApi: undefined,
+      runtimeApi: undefined,
+      error: undefined
+    };
+
+    var start = {type: actType.START_LOAD_RUNTIME};
+    var finish = {type: actType.FINISH_LOAD_RUNTIME, payload: "loadedRuntimeApi"};
+    var fail = {type: actType.FAIL_LOAD_RUNTIME, payload: "reason"};
+
+    it("for action type START_LOAD_RUNTIME", () => {
+      expect(loadApi(state, start).loadingApi).toEqual('started');
+    });
+
+    it("for action type FINISH_LOAD_RUNTIME", () => {
+      expect(loadApi(state, finish).loadingApi).toEqual('finished');
+      expect(loadApi(state, finish).runtimeApi).toEqual(finish.payload);
```

Comment:
Rather than calling loadApi twice, I would just call it once and store the result in a nextState variable. It will make the expect calls easier to read.